fetools-app/src/components/ColorPicker/ColorPickerInterface.jsx

```diff
@@ -49,6 +56,34 @@ export default function ColorPickerInterface({
                 input.value !== inputValues.rgb.b? 
                 input.value = inputValues.rgb.b:
                 input.value = input.value
+            }else if(input.id === 'h'){
+                input.value !== parseInt(inputValues.hsl.h)? 
+                input.value = parseInt(inputValues.hsl.h):
+                input.value = input.value
+            }else if(input.id === 's'){
+                input.value !== parseFloat(inputValues.hsl.s)*100? 
+                input.value = Math.floor(parseFloat(inputValues.hsl.s)*100):
+                input.value = input.value
+            }else if(input.id === 'l'){
+                input.value !== parseFloat(inputValues.hsl.l)*100? 
+                input.value = Math.floor(parseFloat(inputValues.hsl.l)*100):
+                input.value = input.value
+            }else if(input.id === 'c'){
+                input.value !== inputValues.cmyk.c? 
+                input.value = inputValues.cmyk.c:
+                input.value = input.value
+            }else if(input.id === 'm'){
+                input.value !== inputValues.cmyk.m? 
+                input.value = inputValues.cmyk.m:
+                input.value = input.value
+            }else if(input.id === 'y'){
+                input.value !== inputValues.cmyk.y? 
```

Comment:
The logic here is interesting to me.

If it isn't equal to y, set it to y. Otherwise, set it to itself (which at this point we know to be equal to y).

Couldn't you skip the logic here and just set it to y directly?

This assessment is based on a quick read of the code; IDK you have have reason to keep it as is.